web/src/About.svelte

```diff
@@ -5,46 +5,88 @@
 
 <Modal bind:show={$showAbout}>
   <h1>The low-traffic neighbourhood (LTN) tool, v2</h1>
+
   <p>
-    This is an <b>experimental</b>
-    version of the
-    <a
-      href="https://a-b-street.github.io/docs/software/ltn/index.html"
-      target="_blank"
+    This is the second version of the A/B Street LTN tool, <a
+      href="https://github.com/a-b-street/ltn"
+      target="_blank">an open source project</a
+    >. This version, called the Connected Neighbourhoods Tool, has been funded
+    by
+    <a href="https://www.transport.gov.scot/" target="_blank"
+      >Transport Scotland</a
     >
-      A/B Street LTN tool
-    </a>
-    . Most parts of it do not work yet, and you should probably use the other version
-    of the tool instead.
+    and designed by
+    <a href="https://www.sustrans.org.uk" target="_blank">Sustrans Scotland</a>.
+    The team includes:
   </p>
-  <p>To use this tool, you need to:</p>
```

Comment:
This screen isn't the right place for instructions, and the new design of the app makes the flow so much more clear anyway